Remove/Fix outdated paragraphs (#214)
WebAssembly/tool-conventions: EHScheme.md

integrated unwinder. libunwind has a separate library for the unwinder and there
are several implementations including [LLVM's libunwind][llvm_libunwind].

> Our prototype implementation will be based on LLVM's libc++abi and libunwind.

Reviewer: ```suggestion
Our implementation is based on LLVM's libc++abi and libunwind.
```
